kobo/apps/subsequences/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from kpi.models import Asset
+from django.contrib.postgres.fields import JSONField
+
+
+class SubmissionExtras(models.Model):
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_modified = models.DateTimeField(auto_now=True)
+    uuid = models.CharField(max_length=40, null=True)
```

Comment:
Make this refer to submission_uuid (or whatever the right name is) so that I don't assume it's a primary key for SubmissionExtras itself

Reply:
Good point. I will fix this